pkg/pillar/cmd/logmanager/logmanager.go

```diff
@@ -191,11 +191,9 @@ func Run() {
 		log.Fatal(err)
 	}
 
-	logmanagerCtx := logmanagerContext{
-		globalConfig: &types.GlobalConfigDefaults,
-	}
+	logmanagerCtx := logmanagerContext{}
```

Comment:
Hmm.. Without these defaults, we need to wait for GlobalConfig to be published. That will . make sure the agent makes progress only after zedagent actually starts up.